packages/dart/sshnoports/bundles/shell/install.sh

```diff
@@ -154,12 +154,13 @@ install_single_binary() {
   then
     if ! [ -d "$user_bin_dir" ]; then
       mkdir -p "$user_bin_dir"
-      chown -R $user:$user "$user_bin_dir" || chown -R $user "$user_bin_dir"
     fi
 
     if [ -f "$dest/$1" ]; then
       ln -sf "$dest/$1" "$user_bin_dir/$1"
-      chown $user:$user "$user_bin_dir/$1" || chown $user "$user_bin_dir/$1"
+      chown root:wheel "$dest/$1" 2>/dev/null || chown root:root "$dest/$1" 2>/dev/null || chown root "$dest/$1" 2>/dev/null
+      chmod o+x "$dest/$1"
+
```

Comment:
When running as root, root should own the binaries in /usr/local/bin and they should be made executable by others.